src/App.tsx

```diff
@@ -1,12 +1,17 @@
+import Lobby from "@/components/Lobby";
 import "./App.css";
 import GameBoard from "./components/GameBoard";
+import { useState } from "react";
+import { GameConfig } from "@/state/types";
 
-function App() {
-  return (
-    <div className="flex h-full w-full justify-center">
-      <GameBoard />
-    </div>
-  );
-}
+const App: React.FC = () => {
```

Comment:
Any specific reason for the typing here? Our other components dont look like this so I think we should stick to a standard

Reply:
not really. Just some ChatGPT artefacts =)

Reply:
🤖